backend/tracim_backend/lib/core/system.py

```diff
@@ -58,3 +64,14 @@ def get_error_codes(self) -> typing.List[ErrorCodeModel]:
         for error_code in ErrorCode:
             error_codes.append(ErrorCodeModel(error_code))
         return error_codes
+
+    def get_username_availabilities(
```

Comment:
Don't understood why we pass multiple usernames here, can't we just have a method that check avalability for one username ?

Reply:
ok.